src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/prisonapi/PrisonAPI.kt

```diff
@@ -5,6 +5,14 @@ import uk.gov.justice.digital.hmpps.incentivesapi.service.PrisonerInfoForNextRev
 import java.time.LocalDate
 import java.time.LocalDateTime
 
+data class Prison(
+  val agencyId: String,
+  val description: String,
+  val longDescription: String,
```

Comment:
potential danger: this field in nullable (at least according to swagger docs; i didn't check source)

Reply:
Good spot. Even if we don't care for this field for the `KpiService` it could still cause deserialisation issues if some of the prisons have `null` long descriptions. I'll change 👌

Reply:
Changed `Prison` dto class to have optional `longDescription`: https://github.com/ministryofjustice/hmpps-incentives-api/pull/503/commits/9fc99f7f11299240b9a2e7f330d7d9532f109b1b